libs/positions/src/lib/positions-table.tsx

```diff
@@ -21,22 +20,22 @@ interface PositionsTableProps {
 export const getRowNodeId = (data: { market: { id: string } }) =>
   data.market.id;
 
-const sortByName = (
-  a: Positions_party_positions,
-  b: Positions_party_positions
+const comparator = (
+  valueA: string,
+  valueB: string,
+  nodeA: { data: Positions_party_positions },
+  nodeB: { data: Positions_party_positions },
+  isInverted: boolean
 ) => {
-  if (
-    a.market.tradableInstrument.instrument.name <
-    b.market.tradableInstrument.instrument.name
-  ) {
-    return -1;
+  const a = nodeA.data.market.tradableInstrument.instrument.name;
+  const b = nodeB.data.market.tradableInstrument.instrument.name;
```

Comment:
chore (blocking): Many markets may share the same instrument, we likely need a secondary sort. I suggest market by ID as we are guaranteed uniqueness between market/position.

Reply:
Good point. I've added secondary sort on market id